jerry-ext/debugger/debugger-tcp.c

```diff
@@ -131,6 +131,16 @@ jerryx_debugger_tcp_close (jerry_debugger_transport_header_t *header_p) /**< tcp
 
   jerryx_debugger_transport_tcp_t *tcp_p = (jerryx_debugger_transport_tcp_t *) header_p;
 
+  /**
+   * Waiting the remaining tcp data are sent, so that the tcp socket are
+   * closed by debugger client first and the debugger client won't receive
+   * socket are closed by remote error
```

Comment:
I don't understand this sentence.

Reply:
Updated, waiting the debug client close the connection first.